### PR TITLE
make bpkg friendly

### PR DESCRIPTION
**Pull request by jwerle**
This commit adds [bpkg](https://github.com/bpkg/bpkg) and source installation instructions to the `README`.
It also creates a `package.json` file to make installation with `bpkg` simple.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+
+With https://github.com/bpkg/bpkg do:
+
+  $ bpkg install -g eventi/noreallyjustfuckingstopalready
+
+From source:
+
+  $ git clone https://github.com/eventi/noreallyjustfuckingstopalready.git
+  $ cd noreallyjustfuckingstopalready
+  $ install -b flu.sh ${PREFIX:-/usr/local}/bin/flu
```
Comment (eventi) [inline README]: I think you dropped something here

**Comment by eventi** [comment]
Fix up that line and it looks good

File: README (hunk 1 above)
Comment (jwerle) [inline README]: What was dropped?

Comment (eventi) [inline README]: Line 13 should be install -b flu.sh ${PREFIX:-/usr/local}/bin/flu.sh

File: package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "noreallyjustfuckingstopalready",
+  "repo": "eventi/noreallyjustfuckingstopalready",
+  "version": "0.0.1",
+  "description": "Please OS X just fucking reset your DNS cache please",
+  "global": "1",
+  "install": "install -b flu.sh ${PREFIX:-/usr/local}/bin/flu",
```
Comment (eventi) [inline package.json]: same change here

**Comment by jwerle** [comment]
The removal of the `.sh` extension was intentional. This allowed for the simple invocation of `flu`. I can easily change it back though

**Comment by eventi** [comment]
Let's keep in flu.sh - Branding!!

**Comment by jwerle** [comment]
@eventi done !

**Pull request merged**